fix_lock.py

```
@@ -0,0 +1,20 @@
+with open("uv.lock", "r") as f:
+    text = f.read()
+
+import re
```

Flake8 (7.3.0)

[error] 're' imported but unused

(F401)